speakwise/speakwise/speakwise/base/serializers.py

```diff
@@ -0,0 +1,36 @@
+"""Team serializers for the SpeakWise application."""
+
+from rest_framework import serializers
+
+from .models import TeamMember
+
+
+class TeamMemberSerializer(serializers.ModelSerializer):
+    """Serializer for the TeamMember model."""
+
+    avatar_url = serializers.SerializerMethodField()
+
+    class Meta:
+        model = TeamMember
+        fields = [
+            "id",
+            "name",
+            "role",
+            "short_bio",
+            "avatar",
+            "avatar_url",
+            "twitter_url",
+            "linkedin_url",
+            "github_url",
+            "website_url",
+            "display_order",
+        ]
+
+    def get_avatar_url(self, obj):
+        """Get the full URL for the avatar image."""
+        if obj.avatar:
+            request = self.context.get("request")
+            if request:
```

Comment:
Can we write something like this, to ensure that we don't get an error when request is None?
```suggestion
            request = self.context.get("request") if self.context.get("request") else None
```